pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -652,6 +652,12 @@ The max allowed delay for delayed delivery (in milliseconds). If the broker rece
         )
     private int ttlDurationDefaultInSeconds = 0;
 
+    @FieldContext(
+        category = CATEGORY_POLICIES,
+        doc = "System subscriptions that will be ignored by ttl check. Default is empty."
```

Comment:
"Default is empty" in the description, but there's `__compaction` in the default.

Reply:
Please also add changes to the default `broker.conf` and `standalone.conf` files if this is something that should be usually configurable by users.

Reply:
I keep the default value empty and add the changes in `broker.conf` and `standalone.conf`